src/main/kotlin/com/example/todolist/domain/todolist/presentation/data/request/ToDoListRequest.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.todolist.domain.todolist.presentation.data.request
+
+import java.time.LocalDate
+import javax.validation.constraints.NotBlank
+
+data class ToDoListRequest(
+    @field:NotBlank
+    val title:String,
+    @field:NotBlank
+    val content:String,
+    val localDate: LocalDate
```

Comment:
여기도 추후에 date로 변경한다면,
```suggestion
    val date: LocalDate
```
이렇게 변경하는게 좋을거 같습니다.